src/services/dashboard.js

```diff
@@ -38,41 +41,46 @@ export async function queryControllers(params) {
   try {
     const { datastoreConfig, selectedDateRange } = params;
 
-    const endpoint = `${datastoreConfig.elasticsearch}/${getAllMonthsWithinRange(
+    const url = `/${getAllMonthsWithinRange(
```

Comment:
what if we rename this url to `indices` as a URL will mostly refer to the entire request URL?